default.nix

```diff
@@ -289,6 +317,7 @@ in rec {
                       lib.filter (x: lib.hasAttr x self.combinedPackages) self.shells-ghcSavedSplices;
                     ghc = lib.filter (x: lib.hasAttr x self.combinedPackages) self.shells-ghc;
                     ghcjs = lib.filter (x: lib.hasAttr x self.combinedPackages) self.shells-ghcjs;
+                    wasm = lib.filter (x: lib.hasAttr x self.combinedPackages) self.shells-ghcjs;
```

Comment:
`self.shells-ghcjs` should be renamed to `self.shells-frontend` or similar since it now applies to both ghcjs and wasm.